pkg/graph/types.go

```diff
@@ -18,11 +18,12 @@ package graph
 
 import (
 	eventingv1beta3 "knative.dev/eventing/pkg/apis/eventing/v1beta3"
+	"knative.dev/pkg/apis"
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 )
 
 type Graph struct {
-	vertices []*Vertex
+	vertices map[comparableDestination]*Vertex
```

Comment:
`vertices` is a map that contains a bunch of `*Vertex` object, with the key as camparableDestination

Reply:
Yeah, exactly